Cargo.lock

```diff
@@ -7915,6 +7851,15 @@ version = "0.1.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
+[[package]]
+name = "openssl-src"
+version = "300.3.1+3.3.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "7259953d42a81bf137fbbd73bd30a8e1914d6dce43c2b90ed575783a22608b91"
+dependencies = [
+ "cc",
+]
```

Comment:
🤔

Reply:
probably due to https://github.com/rust-lang/cargo/issues/10801

Reply:
Oh, we did use `vendored-openssl`. I think we should use `native-tls` instead?

Reply:
I tried `native-tls` before, but failed to connect to sql server hosted on Azure, but `vendored-openssl` works.

Reply:
Can you paste the errors? Using  `vendored-openssl` will revert our efforts to improve compile speed. https://github.com/risingwavelabs/risingwave/pull/16761

We can do this in follow-up PR though.

Reply:
Will do it in follow-up PR

Reply:
Changed `vendored-openssl` to `rustls` in https://github.com/risingwavelabs/risingwave/pull/18013